### DISCUSSION TITLE
Parameter k for stump and other functions does not exist

### DISCUSSION THREAD

**Discussion by Noskario** [TDAmeritrade/stumpy#885] [General]
By looking at the documentation at https://stumpy.readthedocs.io/en/latest/api.html#stumpy.stump one could get the impression that there is a parameter k that states that the k closest hits should be saved in the matrix profile. However in the implementation '1.11.1' there is no such parameter. Am I missing something? How can I get the matrix profile with k results instead of one?

**Comment by seanlaw** [comment]
@Noskario Thank you for your question and apologies in advance for the confusion. Currently, our documentation reflects the development version of our code and so the `k` parameter is only available if you install stumpy from source:

```
python -m pip install git+https://github.com/TDAmeritrade/stumpy.git
```

A new release (with the `k` parameter) will be released soon!

**Comment by Noskario** [comment]
Thank You for Your help, this solved my issue.
